### ISSUE TITLE
Tooltip not centered

### ISSUE THREAD

**Issue by i-schuetz** [skydoves/Balloon#28]
**Please complete the following information:**
- Library Version 1.1.2
- Affected Device(s) Emulator
 
**Describe the Bug:**

```kotlin
packageEdit.setOnClickListener {
    packageTitle.showBalloon(this)
}
```
`packageTitle` is the label where it should appear centered.
Settings:
```kotlin
setArrowSize(10)
//        setWidthRatio(1.0f)
setHeight(65)
setPadding(0)
setDismissWhenTouchOutside(true)
setPaddingLeft(10)
setPaddingRight(10)
//        setArrowPosition(.5f)
setSpace(10)
setCornerRadius(10f)
setAlpha(1f)
setTextColorResource(R.color.white)
setBackgroundColorResource(R.color.colorAccent)
setOnBalloonClickListener {
    onClick?.invoke()
}
setBalloonAnimation(BalloonAnimation.ELASTIC)
setLifecycleOwner(lifecycleOwner)
```
Happens when using in `RecyclerView`. Position seems to be random, on other cells it appears at different non-centered positions. I haven't tested without `RecyclerView`.

<img width="396" alt="Screen Shot 2020-03-24 at 15 53 33" src="https://user-images.githubusercontent.com/1381744/77439753-bb91f800-6de7-11ea-913c-87a1ce7064f8.png">

**Expected Behavior:**

The tooltip appears centered horizontally over the text label.

**Comment by skydoves** [comment]
#31

**Comment by skydoves** [comment]
The measuring isn't always correct because of the Android SDK's problem.
So you should make a clear position using `x-off` and `y-off` when showing a balloon.

**Comment by i-schuetz** [comment]
Which problem exactly? I could write code to measure the label's width and set `x-off` accordingly, but it doesn't seem something that the library can't do directly?

**Comment by skydoves** [comment]
@i-schuetz 
Hi, I found the problem, it will be fixed in the next version. :)
Thank you for your issue!

**Comment by i-schuetz** [comment]
@skydoves Awesome, thank you! Would this be `1.1.3`?

**Comment by skydoves** [comment]
It will be released on 1.1.4 today. 
Thank you :)

**Comment by skydoves** [comment]
@i-schuetz 
Hi, it is released a new version `1.1.4`.
You can check now!

**Comment by i-schuetz** [comment]
It's working now, thanks!
